flex-panel-gallery/style.css

```diff
@@ -0,0 +1,86 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+html {
+  font-size: 10px;
+}
+body {
+  background-color: #fb6707;
+}
+.panels {
+  height: 99.9vh;
+  width: 99.9vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: "Poppins", sans-serif;
+  color: #ffffff;
+}
+
+.panel {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  height: 100%;
+  font-size: 8rem;
+  transition: flex 4s cubic-bezier(0.61, -0.19, 0.7, -0.11),
+    font-size 2s cubic-bezier(0.61, -0.19, 0.7, -0.11);
+  overflow: hidden;
+}
+.panel1 {
+  background: url(./pictures/bridge.jpg) no-repeat;
+  background-size: cover;
```

Comment:
make sense to add repeated properties to .panel

Reply:
Sorry, I muse have confused you. 
What I wanted to say, that each .panel{n} class has the same properties, like
`background-size: cover;` and `background-position: center;` and this duplicates can be added to the .panel class